app/models/meetup.server.test.ts

```diff
@@ -46,19 +49,23 @@ describe("meetup.server", () => {
       );
     });
 
+    afterEach(() => {
+      lastRequestBody = undefined;
+    });
+
     it("should call meetup api", async () => {
       const event = await tryToFetchRemixAustinInfo();
       expect(lastRequestBody).toBeDefined();
```

Comment:
One small suggestion here: I like to use `invariant` from `tiny-invariant` for these kinds of checks, since if it errors, that will cause the test to fail with a good message. This allows you to get rid of the optional chaining.

Definitely not a big thing, just a pattern I personally like

```ts
invariant(lastRequestBody, "lastRequestBody must be defined");
expect(lastRequestBody.variables).toEqual({ urlname: "remix-austin" });
// TODO use gql utils for better assertions
expect(lastRequestBody.query.includes("groupByUrlname")).toBeTruthy();
```

Reply:
interesting strategy. it def makes the err msg better and avoids the optional chaining, but does read a little weird since you end up with a mixture of `invariant`s and `expect`s. 🤔 initial thought is the pros outweigh the con there. but either way, i'll match your style.